frontend/src/metabase/search/components/SearchSidebar/SearchSidebar.tsx

```diff
@@ -1,59 +1,104 @@
 import _ from "underscore";
+import { useCallback, useMemo } from "react";
 import type {
   FilterTypeKeys,
+  SearchFilterComponent,
   SearchFilterPropTypes,
   URLSearchFilterQueryParams,
-  SearchSidebarFilterComponent,
 } from "metabase/search/types";
 import { Stack } from "metabase/ui";
 import { SearchFilterKeys } from "metabase/search/constants";
-import { SidebarFilter } from "metabase/search/components/SidebarFilter/SidebarFilter";
+import { DropdownSidebarFilter } from "metabase/search/components/SearchSidebar/DropdownSidebarFilter/DropdownSidebarFilter";
 import { TypeFilter } from "metabase/search/components/filters/TypeFilter/TypeFilter";
+import { PLUGIN_CONTENT_VERIFICATION } from "metabase/plugins";
+import { ToggleSidebarFilter } from "metabase/search/components/SearchSidebar/ToggleSidebarFilter/ToggleSidebarFilter";
 import { CreatedByFilter } from "metabase/search/components/filters/CreatedByFilter/CreatedByFilter";
 
 type SearchSidebarProps = {
   value: URLSearchFilterQueryParams;
   onChange: (value: URLSearchFilterQueryParams) => void;
 };
 
-export const filterMap: Record<FilterTypeKeys, SearchSidebarFilterComponent> = {
-  [SearchFilterKeys.Type]: TypeFilter,
-  [SearchFilterKeys.CreatedBy]: CreatedByFilter,
-};
-
 export const SearchSidebar = ({ value, onChange }: SearchSidebarProps) => {
+  const filterMap: Record<FilterTypeKeys, SearchFilterComponent | null> =
+    useMemo(
+      () => ({
+        [SearchFilterKeys.Type]: TypeFilter,
+        [SearchFilterKeys.CreatedBy]: CreatedByFilter,
+        [SearchFilterKeys.Verified]: PLUGIN_CONTENT_VERIFICATION.VerifiedFilter,
+      }),
+      [],
+    );
+
+  const isValidFilterValue = useCallback(
+    (
+      key: FilterTypeKeys,
+      val: SearchFilterPropTypes[FilterTypeKeys],
```

Comment:
can you use the unabbreviated version `value` instead? We try not to abbreviate names as much as possible to make everything as clear as possible.

edited: I noticed this was kind of needed because we also have another variable called `value`, but to be fare, both names sound pretty similar but have totally different shapes. I think we should differentiate them more e.g.
- `value` -> `filters` since this is a map that contains all the filters.
- `val` -> `value` since this is filter values e.g. string or array